src/feelpp/benchmarking/configSchema.json

```diff
@@ -0,0 +1,133 @@
+                                "prefix": { "type": "string" },
+                                "case": { "type": "string" }
+                            },
+                            "required": ["prefix", "case"]
+                        },
+                        "case": {
+                            "type": "object",
+                            "properties": {
+                                "dimension": { "type": "string" },
+                                "discretization": { "type": "string" }
+                            }
+                        },
+                        "jsonPatch": {
+                            "oneOf": [
+                                {
+                                    "type": "array",
+                                    "items": {
+                                        "type": "object",
+                                        "properties": {
+                                            "op": { "type": "string" },
+                                            "path": { "type": "string" },
+                                            "value": {
+                                                "oneOf": [
+                                                    { "type": "string" },
+                                                    { "type": "number" }
+                                                ]
+                                            }
+                                        },
+                                        "required": ["op", "path", "value"]
+                                    }
+                                },
+                                {
+                                    "type": "object",
+                                    "properties": {
+                                        "op": { "type": "string" },
+                                        "path": { "type": "string" },
+                                        "value": {
+                                            "oneOf": [
+                                                { "type": "string" },
+                                                { "type": "number" }
+                                            ]
+                                        }
+                                    },
+                                    "required": ["op", "path", "value"]
+                                }
+                            ]
+                        }
+                    },
+                    "required": ["config-files", "repository", "jsonPatch"]
+                }
+            },
+            "required": ["toolbox", "CommandLine"]
+        },
+        "Reporter": {
+            "type": "object",
+            "properties": {
+                "active": { "type": "boolean" },
+                "scaling": { "type": "string" }
+            },
+            "required": ["active", "scaling"]
+        }
+    },
+    "required": ["Reframe", "Feelpp", "Reporter"]
+}
```

Comment:
use github best practice for naming convention
https://github.com/saifaustcse/api-best-practices